homeassistant/components/androidtv/media_player.py

```diff
@@ -133,7 +133,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Android TV / Fire TV platform."""
     hass.data.setdefault(ANDROIDTV_DOMAIN, {})
 
-    host = f"{config[CONF_HOST]}:{config[CONF_PORT]}"
+    address = f"{config[CONF_HOST]}:{config[CONF_PORT]}"
```

Comment:
The `androidtv` package now accepts the `host` and `port` parameters separately, so I renamed this variable.